apps/liquidity-provision-dashboard/src/app/components/dashboard/market-list/market-list.tsx

```diff
@@ -50,7 +50,11 @@ export const MarketList = () => {
         <Grid
           gridOptions={{
             onRowClicked: ({ data }: RowClickedEvent) => {
-              navigate(`/markets/${data.id}`);
+              window.open(
```

Comment:
why can't we use `navigate` ?

Reply:
Good question!

My understanding is that `useRouter` and thus `navigate` are designed for _internal_ routing changes, which is why I previously added them here to handle navigating to the details page. Now that we're replacing the _internal_ details page with a link to the _external_ console liquidity details page - it didn't make sense to me to try and use `useRouter`/`navigate` - my understanding was that external links were unsupported behaviour (even if they might work) ?

Reply:
@MadalinaRaicu does this make sense? What do you reckon - stick with this commit or try and make it work with `navigate`/`useRouter` ?